src/airport.js

```
@@ -1,6 +1,63 @@
+const Plane = require(`./Plane`);
+const Weather = require("./Weather");
+
 class Airport {
-  // here's a starting point for you
 
+    // In future I would make these properties private
+    planesLanded = [];
+    capacity = 10;
+    weather;
+    name;
+
+    // Each airport receives its own named instance of weather, or a default instance. 
+    // Would be better as a static method in future
+    constructor(weather, name) {
+        this.weather = weather;
+        this.name = name;
+    }
+
+    isItStormy = () => this.weather?.isItStormy();
+
+    // These would be better throwing errors, rather than a log
+    landPlane = plane => {
+        if (this.isItStormy())
+            return console.log(`Landing is not allowed in a storm`);
```

This should probably throw an error instead of logging to the console